pom.xml

```diff
@@ -641,6 +641,22 @@ under the License.
   </dependencies>
   <!--bootstrap-end-comment-->
 
+  <repositories>
```

Comment:
Should we get rid of this now ? 
```
➜  maven git:(model-buider-v4-b) ✗ mima list org.apache.maven.resolver:maven-resolver-connector-basic | grep 2.0.0-alpha-10
org.apache.maven.resolver:maven-resolver-connector-basic:2.0.0-alpha-10
```

Reply:
```
[cstamas@angeleyes ~]$ jbang mima@maveniverse exists org.apache.maven.resolver:maven-resolver-api:2.0.0-alpha-10
Artifact org.apache.maven.resolver:maven-resolver-api:jar:2.0.0-alpha-10 NOT EXISTS

Checked TOTAL of 1 (existing: 0 not existing: 1)
[cstamas@angeleyes ~]$ 
```

Reply:
List GA uses _maven metadata_, that is on other synced somehow...